ci-scripts/yocto2.groovy

```diff
@@ -66,7 +66,14 @@ void buildImageAndSDK(String yoctoDir, String imageName, String variantName, boo
     } else {
         echo "\'site.conf\' exists, using cache"
     }
-
+    stage("Verify NFS") {
+        def sC = sh script:"mount | grep -w nfs4", returnStatus:true
```

Comment:
Would it not be better to check path here instead of "any NFS fs is mounted"?

Reply:
Or perhaps just check if the "downloads" folder and "sstate-cache" folder exist. Don't involve "mount" at all. Script is even less specific to our particular setup.

Reply:
Do you mean "mount | grep -w '/var/yocto-cache'"?

Reply:
> Or perhaps just check if the "downloads" folder and "sstate-cache" folder exist. Don't involve "mount" at all. Script is even less specific to our particular setup.

It's not thee best idea, as those directories can be presented locally. Another option is to fail build  if NFS is not mouned (read "dirs sstate-cache and downloads are absent")

Reply:
Yes, that would make it so this test does not give a false positive if another NFS mount exists on the system.

But also, why require a mount at all. If one wants to not use a mount (NFS or other one, the way we had it before NFS was setup) for some reason, using mount to check would make this warning always trigger. So why not just do what I mentioned in second comment? I.e. just check that  `${yoctoCacheArchivePath}/sstate-cache` and `${yoctoCacheArchivePath}/downloads` exist.